### PR TITLE
apps/plans/forms//modal//plan_form: add form field to modal, adapt in…

### PR DESCRIPTION
**Pull request by phillimorland**
… form and add to template -no migrations

### CODE REVIEW

File: meinberlin/apps/plans/forms.py [hunk 1]
```diff
@@ -49,6 +50,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['district'].empty_label = _('City wide')
         self.fields['contact_address_text'].widget.attrs['rows'] = 6
+        self.fields['participation_explanation'].widget.attrs['rows'] = 1
```
Comment (Rineee) [inline meinberlin/apps/plans/forms.py:53]: as I understood, it should look like address field, so also 6 rows here?

Comment (phillimorland) [reply meinberlin/apps/plans/forms.py:53]: address field reference was just to ensure linebreaks so it should actually go in the user side story, in designs it is just one line but can be expanded

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! 
I will fix the tests!

**Comment by phillimorland** [comment]
@fuzzylogic2000 ahh right tests, thanky you!

**Pull request merged**